create_mountaineer_app/create_mountaineer_app/__tests__/test_builder.py

```diff
@@ -6,7 +6,7 @@
 from uuid import uuid4
 
 import pytest
-import tomllib
+import toml
```

Comment:
Since the API scope of what we use between tomllib and toml is the same (should just be a simple `.loads`), better to keep dependencies slim on modern devices. Can we make this a conditional that only imports toml if tomllib isn't found on the local OS?